File: python/problem_10.py
from helpers import find_primes, upper_bound_for_nth_prime


def summation_of_primes(limit=2000000):
    return sum([prime if prime < limit else 0 for prime in list(find_primes(limit))])


print(True if summation_of_primes() == 142913828922 else False)

File: php/problem_10.php
<?php
include "helpers.php";

function summation_of_primes($limit=2000000, $result=0)
{
  $primes_generator = find_primes($limit);
  $primes = [];
  foreach ($primes_generator as $prime) {
    if ($prime < $limit) {
      $primes[] = $prime;
    }
  }
  return array_sum($primes);
}

summation_of_primes();